avm/res/compute/gallery/image/main.bicep

```diff
@@ -256,3 +155,120 @@ type roleAssignmentType = {
   @sys.description('Optional. The Resource Id of the delegated managed identity resource.')
   delegatedManagedIdentityResourceId: string?
 }[]?
+
+@export()
```

Comment:
While I applaud the idea in order to save code, I strongly advice against wrapping the entire properties block. 
Instead, I'd advice to create one UDT per non-top level property that is either an array or object (like the `identifier` property), mark them all via `export()` and use them in a UDT in the parent.

Reply:
I've changed the parameters after our conversation.